Lib/shutil.py

```diff
@@ -527,12 +530,11 @@ def _copytree(entries, src, dst, symlinks, ignore, copy_function,
                 copytree(srcobj, dstname, symlinks, ignore, copy_function,
                          ignore_dangling_symlinks, dirs_exist_ok)
             else:
-                # Will raise a SpecialFileError for unsupported file types
                 copy_function(srcobj, dstname)
-        # catch the Error from the recursive copytree so that we can
-        # continue with other files
+        except ErrorGroup as err_group:
+            errors.extend(err_group.args[0])
         except Error as err:
-            errors.extend(err.args[0])
+            errors.append((srcname, dstname, str(err)))
```

Comment:
`Error` extends `OSError`, so there is no need to catch this any more.

Reply:
Yup. Fixed!